docs/DeveloperGuide.adoc

```diff
@@ -279,13 +457,15 @@ Refer to the guide <<DevOps#, here>>.
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing over mouse input
-* is reasonably comfortable using CLI apps
+* Enjoys exercising and does it regularly
+* Has a need to manage a significant number of workouts
+* Prefers desktop apps over other types
+* Can type fast
+* Prefers typing over mouse input
+* Is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*:
+Manage workouts faster than a typical mouse/GUI-driven app
 
 [appendix]
 == User Stories
```

Comment:
Make sure all user stories are grammatically correct. I see some inconsistencies.

As a user......., I want TO ......., so that I can .......